index.js

```diff
@@ -0,0 +1,86 @@
+/* eslint-env browser */
+/* global _gaq, ga */
+import { on, find } from 'domassist';
+import aug from 'aug';
+
+class GATrack {
```

Comment:
I think you're right. We shouldn't create a class. I did it at first cause I wasn't sure but then I should have removed it.

The el is only to track one given element, specifically. You shouldn't mess normally with the library as autotrack is called on page load and that's enough. Calling autotrack again will pick any new elements and ignore the ones already defined.

Let me know if this makes sense.